esmvalcore/_recipe.py

```diff
@@ -108,7 +108,7 @@ def _add_cmor_info(variable, override=False):
         raise RecipeError(
             f"Unable to load CMOR table (project) '{project}' for variable "
             f"'{short_name}' with mip '{mip}'")
-
+    variable['original_short_name'] = table_entry.short_name
```

Comment:
I know that there may not be an easy way around this, but this adds yet another name for the variable, thus we will have for every variable:
- cmor_name
- short_name
- original_short_name
- standard_name
- long_name

this is becoming more and more confusing. Maybe we could avoid introducing this extra name somehow?

Reply:
We use it in the data_finder, to look for the actual name in the files not the requested and on the save to decide if we need to rename the var_name. Let me think if I can find another way of doing those things without this new variable

Reply:
Maybe we could try to expose this as little as possible, e.g. only set `original_short_name` for variables where it differs from the `short_name` and not pass it on to the diagnostic script?

Reply:
if it's completely under the hood then I'm fine with it, otherwise if the user will have to see it and set it/change it, then I'm with @bouweandela on this one :beer:

Reply:
It is completely internal, in fact that is the whole point of managing the alias and renaming the short_names, that the diagnostics do not care about equivalent names of variables and just see the name they requested in the recipe

Reply:
I just tried and the `original_short_name` ends up in metadata.yml, making it visible to diagnostic developers. Could you make sure it doesn't end up in there?

Reply:
Should be fixed now